packages/@adobe/spectrum-css-temp/components/colorarea/index.css

```diff
@@ -16,14 +25,7 @@
   &.focus-ring {
     z-index: 2;
 
-    .spectrum-ColorArea-handle {
-      /* Bigger handle when focused */
-      width: calc(var(--spectrum-colorhandle-size) * 2);
-      height: calc(var(--spectrum-colorhandle-size) * 2);
-
-      margin-left: calc(-1 * var(--spectrum-colorhandle-size));
-      margin-top: calc(-1 * var(--spectrum-colorhandle-size));
-    }
+    @extend %spectrum-ColorControl-handle--focused;
```

Comment:
should just be able to do
```
.spectrum-ColorArea-handle.spectrum-ColorArea-handle
```
it looks silly, but it will raise the specificity

Reply:
Raising the specificity doesn't seem to make a difference without the other changes in this PR.